notebooks/executing-notebooks-from-another-notebook-with-fusion-sql/notebook.ipynb

```diff
@@ -0,0 +1,306 @@
+      ],
+      "id": "b04db0d5"
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Next, we execute the notebook and confirm that the `sample_var` variable set in the sample notebook is accessible in the current session:"
+      ],
+      "id": "9c08ff46"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 4,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "if 'sample_var' in globals():\n",
+        "    del sample_var\n",
+        "\n",
+        "%run sample_notebook.ipynb\n",
+        "\n",
+        "print(\"The value of 'sample_var' is '{}'.\\n\".format(sample_var))"
+      ],
+      "id": "85ad1f06"
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Executing the Sample Notebook in a New Session\n",
+        "\n",
+        "Instead of executing a notebook in the current session, we can run it in a new session \u2014 either synchronously or asynchronously \u2014 using jobs. This approach is useful for parallel execution or running code in a separate runtime environment.\n",
+        "\n",
+        "<div class=\"alert alert-block alert-warning\">\n",
+        "    <b class=\"fa fa-solid fa-exclamation-circle\"></b>\n",
+        "    <div>\n",
+        "        <p><b>Note</b></p>\n",
+        "        <p>SingleStore enforces <a href=\"https://docs.singlestore.com/cloud/developer-resources/notebooks/using-notebooks/#rate-limits\">limits</a> on the number of compute sessions that can be created, which can be increased upon request. Please contact <a href=\"https://support.singlestore.com/hc/en-us\">Support</a> for more information.</p>\n",
+        "    </div>\n",
+        "</div>"
+      ],
+      "id": "1a059fcb"
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Run **two jobs** in parallel and wait for their completion (each job will run on a separate session):"
+      ],
+      "id": "b5d35764"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "job_ids = []\n",
+        "for x in range(2):\n",
+        "    print(\"Running job for {}...\".format(x))\n",
+        "    job_res = %sql RUN JOB USING NOTEBOOK '{{ sample_notebook_name }}' WITH PARAMETERS {\"sample_var\": \"{{x}}\"}\n",
+        "    job_ids.append(job_res[0].JobID)\n",
+        "\n",
+        "print(f'Waiting for jobs to complete... {job_ids}')\n",
+        "success = %sql WAIT ON JOBS {{ job_ids }} WITH TIMEOUT 60 MINUTES\n",
+        "\n",
+        "print(f'All jobs completed with success: {bool(success[0].Success)}')"
+      ],
+      "id": "2cdaea66"
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "View the executions of the jobs we run:"
```

Comment:
`run` -> `ran`

Reply:
Thank you for pointing out that typo; I have already fixed it!